bindings/wallet-js/src/lib.rs

```diff
@@ -297,18 +297,6 @@ impl Ed25519Signature {
     }
 }
 
-#[wasm_bindgen]
-pub fn symmetric_encrypt(password: &[u8], data: &[u8]) -> Result<Box<[u8]>, JsValue> {
```

Comment:
Take care to bump the pre-release version in wallet-js as these removals change the API. Perhaps do it in this PR just as well.
Luckily we can still do this without violating semver or having to increase the minor version number just for this.

Reply:
Done.

I'm actually bumping versions in all the wallet crates with each release anyway, otherwise the crate versions wouldn't really match with the tags, although it's not really the cleanest thing ever.

Also, we should probably make a non pre-release soon.